src/server/pfs/server/driver.go

```diff
@@ -157,8 +157,40 @@ func newDriver(env *serviceenv.ServiceEnv, etcdPrefix string, treeCache *hashtre
 	return d, nil
 }
 
+type Operation interface {
+	validate(d *driver, pachClient *client.APIClient) error
+	execute(d *driver, pachClient *client.APIClient, stm col.STM) error
+}
+
+type CreateBranchOp struct {
+	*pfs.CreateBranchRequest
+}
```

Comment:
I think you can do this a bit more tersely by doing:

```golang
type CreateBranchRequest pfs.CreateBranchRequest
```
Basically just lifting the type into this file so you can define methods on it. I also think we're better off just sticking with the word "Request" for everything and not introducing a new word "Op", validating and executing a request both seem like totally reasonable things to do, so I think the language fits.